src/provider/AbstractUnixProvider.php

```diff
@@ -9,6 +9,7 @@
 abstract class AbstractUnixProvider extends AbstractProvider
 {
     protected $cpuInfo;
+	protected $cpuInfoByLsCpu;
```

Comment:
I could ask the same question about `cpuInfo`. Why is that in `AbstractUnixProvider` when corresponding methods are also in `LinuxProvider`?